docs/csharp/language-reference/configure-language-version.md

```diff
@@ -20,6 +20,17 @@ You can override the default behavior by specifying a language version. There ar
 - Set the language version [for multiple projects in a subdirectory](#configure-multiple-projects).
 - Configure the [`-langversion` compiler option](#set-the-langversion-compiler-option).
 
+## Required SDK and Runtime version
+
+Language features may rely on library types, or APIs that are introduced in specific versions of .NET Standard, or specific implementations of the .NET library. The following list provides information on the supported SDK version corresponding to recent language versions:
+
+- **C# 8.0**: *netstandard2.1*, including *netcoreapp3.0*.
+- **C# 7.3**: *netstandard2.0*, including *netcoreapp2.1*, and *.NET Framework 4.7*.
+- **C# 7.2**: *netstandard2.0*, including *netcoreapp2.1*, and *.NET Framework 4.7*.
+- **C# 7.1**: *netstandard2.0*, including *netcoreapp2.1*, and *.NET Framework 4.7*.
+- **C# 7.0**: *netstandard2.0*, including *netcoreapp2.1*, and *.NET Framework 4.7*.
+- **C# 6.0**: *netstandard1.3*, including *netcoreapp1.0*, and *.NET Framework 4.6*.
```

Comment:
Why is this list mixing TFMs and a framework name? Shouldn't it be either all TFMs (e.g. netstandard1.3, netcoreapp1.0 and net46) or, even better, all proper names (e.g. .NET Standard 1.3,  .NET Core 1.0 and .NET Framework 4.6).

Though I guess .NET Core x.y is unclear, since it could mean either SDK or runtime.

Reply:
The proper names are a better choice. I went with that.

I think this reads more clear by referencing TFMs only. From there, I think most users can infer:

1. You must use a version of the SDK that supports that TFM.
1. You must have run on a runtime that includes the TFM you built against.